### DISCUSSION TITLE
I can't find estimated duration from workflow progress info.

### DISCUSSION THREAD

**Discussion by purin0604** [argoproj/argo-workflows#7566] [Q&A] [answered]
It seems like Argo Workflow provides estimated durations showing in its UI or in the workflow progress.
(https://github.com/argoproj/argo-workflows/pull/4015 ,  https://argoproj.github.io/argo-workflows/estimated-duration/)
However, I can only see the current duration as follows.

```
Name:                parallel-job10-7h277
Namespace:           argo
ServiceAccount:      default
Status:              Running
Conditions:          
 PodRunning          False
Created:             Fri Jan 14 23:28:26 -0800 (10 seconds ago)
Started:             Fri Jan 14 23:28:26 -0800 (10 seconds ago)
Duration:            10 seconds
Progress:            1/1
ResourcesDuration:   5s*(1 cpu),9s*(100Mi memory)
```

Is there any command should I use to show "Estimated Duration" in the progress?

**Comment by terrytangyuan** [comment, marked as answer]
> This is based on the most recently successful workflow submitted from the same workflow template, cluster workflow template or cron workflow.

You need to submit from the same workflow templates to obtain the estimated duration from a recent successful workflow.

**Comment by purin0604** [comment]
@terrytangyuan I have tried to submit the same workflow templates several times on the same day, but still Argo UI doesn't show the estimated duration. Does it work only for the local image or particular type of images? I tested it with alpine images.

**Reply by terrytangyuan** [reply]
If that's the case, could you submit a bug report instead and fill out some necessary information that would help debugging?
